CHANGELOG.md

```diff
@@ -4,6 +4,41 @@
 
 ## [Unreleased (beta)]
 
+## [0.13.0] - 2018-06-20
+
+Release notes: https://github.com/DataDog/dd-trace-rb/releases/tag/v0.13.0
+
+Git diff: https://github.com/DataDog/dd-trace-rb/compare/v0.12.1...v0.13.0
+
+### Added
+
+- Sequel integration (supporting Ruby 2.0+) (#171, #367) (@randy-girard, @twe4ked, @palin)
+- gRPC integration (supporting Ruby 2.2+) (#379, #403) (@Jared-Prime)
+- ActiveModelSerializers integration (#340) (@sullimander)
+- Excon integration (#211) (@walterking)
+- Rake integration (supporting Ruby 2.0+, Rake 12.0+) (#409)
+- Request queuing tracing to Rack (experimental) (#272)
+- ActiveSupport::Notifications::Event helper for event tracing (#400)
+- Request and response header tags to Rack (#389)
+- MySQL2 integration (#453) (@jamiehodge)
+- Sidekiq tag job delay (#458) (@gottfrois)
```

Comment:
This has the wrong PR #458 should be #443. This also needs to list #418.

Reply:
😱 Thanks for spotting that !